test/d-b-n_smoke/reconnection.html

```diff
@@ -63,8 +66,15 @@
 
             it('when connection is broken with a reason should display an error box', function () {
                 var closeEvent = {reason: 'thereason', code:1};
+                var boxErrorMsg = 'Connection error. See console for details.';
+                puppet.network._ws.close(closeEvent);
+                expect(container.querySelector("a.box.error.reload").innerText).to.contain(boxErrorMsg);
+            });
+
+            it('and log error reason to the console', function() {
+                var closeEvent = {reason: 'thereason', code: 1};
                 puppet.network._ws.close(closeEvent);
-                expect(container.querySelector("div.box.error").innerText).to.contain(closeEvent.reason);
+                expect(consoleErrorSpy.getCall(0).args[0]).to.contain(closeEvent.reason);
             });
 
             describe('when connection is broken without a reason should display a notice box', function() {
```

Comment:
I was not able to run `wct` locally (seems to freeze for me), but running `http://localhost:8000/components/puppet-polymer-client/test/` in web browsers (Chrome, Firefox, Edge) shows 100% passing tests 👍